### PR TITLE
[ele] fix lasso

### PR DESCRIPTION
**Pull request by HawkCorrigan**

### CODE REVIEW

File: engine/class_modules/sc_shaman.cpp [hunk 1]
```diff
@@ -8252,24 +8252,42 @@ struct healing_stream_totem_spell_t : public shaman_totem_t<heal_t, shaman_heal_
...
   }
+  double composite_persistent_multiplier( const action_state_t* state ) const override
+  {
+    double m = shaman_spell_t::composite_persistent_multiplier( state );
+    if ( p()->buff.master_of_the_elements->up() )
+    {
+      m *= 1.0 + p()->buff.master_of_the_elements->default_value;
+    }
+      return m;
+  }
```
Comment (Bloodmallet) [inline engine/class_modules/sc_shaman.cpp]: is this section necessary, if you set `affected_by_master_of_the_elements = true` above?

File: engine/class_modules/sc_shaman.cpp [hunk 2]
```diff
@@ -8252,24 +8252,42 @@ struct healing_stream_totem_spell_t : public shaman_totem_t<heal_t, shaman_heal_
...
-    tick_may_crit      = true;
-    may_crit           = false;
+    affected_by_master_of_the_elements = true;
+    cooldown->duration                 = p()->find_spell( 305483 )->cooldown();
+    trigger_gcd                        = p()->find_spell( 305483 )->gcd();
+    channeled                          = true;
+    tick_may_crit                      = true;
+  }
+
+  bool ready()override
```
Comment (Bloodmallet) [inline engine/class_modules/sc_shaman.cpp]: please add a space between `()override`

File: engine/class_modules/sc_shaman.cpp [hunk 3]
```diff
@@ -8252,24 +8252,42 @@ struct healing_stream_totem_spell_t : public shaman_totem_t<heal_t, shaman_heal_
 // PvP talents/abilities
 // ==========================================================================
 
-struct lightning_lasso_t : public shaman_spell_t
+ struct lightning_lasso_t : public shaman_spell_t
 {
-  lightning_lasso_t( shaman_t* player, util::string_view options_str ) :
-    shaman_spell_t( "lightning_lasso", player, player->talent.lightning_lasso )
+  lightning_lasso_t( shaman_t* player, util::string_view options_str ) :
```
Comment (Bloodmallet) [inline engine/class_modules/sc_shaman.cpp]: Could you remove your additional spaces?

**Review by Bloodmallet — CHANGES_REQUESTED** [review]

File: engine/class_modules/sc_shaman.cpp (hunk 1 above)
Comment (HawkCorrigan) [reply engine/class_modules/sc_shaman.cpp]: yes

**Pull request merged**